src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,64 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
```

Comment:
Consider specifying the `fetch` and `cascade` attributes for the `@OneToMany` relationship. By default, `fetch` is `LAZY`, which might not be suitable if you need to access tickets frequently. Also, consider using `cascade = CascadeType.ALL` if you want changes to the `Order` to propagate to the associated `Ticket` entities.